### DISCUSSION TITLE
How to use react-native-reanimated with react-native-svg-transformer?

### DISCUSSION THREAD

**Discussion by iamrohitagg** [software-mansion/react-native-reanimated#6797] [Q&A]
I am trying to change my metro.config.js file for the [third step](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started/#step-3-wrap-metro-config-with-reanimated-wrapper-recommended) as recommended. But i have already changed my metro.config.js for [react-native-svg-transformer](https://www.npmjs.com/package/react-native-svg-transformer), and if I do some changes the svg package does not work for me.
<br>
Here is my metro.config.js
```const { getDefaultConfig, mergeConfig } = require("@react-native/metro-config");
const {
  wrapWithReanimatedMetroConfig,
} = require("react-native-reanimated/metro-config");

/**
 * Metro configuration
 * https://facebook.github.io/metro/docs/configuration
 *
 * @type {import('metro-config').MetroConfig}
 */
const config = async () => {
  const defaultConfig = getDefaultConfig(__dirname);
  const {
    resolver: { sourceExts, assetExts },
  } = defaultConfig;

  return mergeConfig(defaultConfig, {
    transformer: {
      babelTransformerPath: require.resolve("react-native-svg-transformer"),
      getTransformOptions: async () => ({
        transform: {
          experimentalImportSupport: false,
          inlineRequires: true,
        },
      }),
    },
    resolver: {
      assetExts: assetExts.filter((ext) => ext !== "svg"),
      sourceExts: [...sourceExts, "svg"],
    },
  });
};

module.exports = config;
```

**Comment by MatiPl01** [comment]
Hey @iamrohitagg!

Sorry for the late response. Your configuration doesn't work if you add `wrapWithReanimatedMetroConfig` here?
```js
module.exports = wrapWithReanimatedMetroConfig(config);
```

Can you paste the error message you get?

Addint `wrapWithReanimatedMetroConfig` is recommended but not required so you can skip this step if it is problematic.
